### ISSUE TITLE
Shift tabbing back past the HeaderSearch isn't possible

### ISSUE THREAD

**Issue by brudil** [sussexstudent/website#239]

**Comment by brudil** [comment]
Should be fixed in Loki as we're not doing anything too crazy now. See #344
